From c8e26099678c531f7f2539af1ac6ee44fae4365b Mon Sep 17 00:00:00 2001
From: Kevin Ryde <user42_kevin@yahoo.com.au>
Date: Sun, 25 Aug 2019 10:14:42 +0200
Subject: [PATCH] contrib/fetchmail-mode.el: run fetchmail-mode-hook after
 other settings

Obtained from: https://bugs.debian.org/710319
Obsoletes: Debian patch 02_fetchmail_mode.el.diff
---
 contrib/fetchmail-mode.el | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/contrib/fetchmail-mode.el b/contrib/fetchmail-mode.el
index d22ebcaf..a9038810 100644
--- a/contrib/fetchmail-mode.el
+++ b/contrib/fetchmail-mode.el
@@ -126,7 +126,6 @@
   (use-local-map fetchmail-mode-map)    ; This provides the local keymap.
   (setq mode-name "Fetchmail")          ; This name goes into the modeline.
   (setq major-mode 'fetchmail-mode)     ; Used by `describe-mode'
-  (run-hooks 'fetchmail-mode-hook)      ; Run each time mode is called
   (set-syntax-table fetchmail-mode-syntax-table)
 
   ;; -cc-
@@ -134,7 +133,8 @@
   (make-local-variable 'font-lock-defaults)
   (setq font-lock-defaults '(fetchmail-font-lock-keywords nil t))
 
-  (setq comment-start "#")
+  (set (make-local-variable 'comment-start) "#")
+  (run-hooks 'fetchmail-mode-hook)      ; Run each time mode is called, last
   )
 
 
-- 
2.22.0

